src/style.css

```diff
@@ -0,0 +1,20 @@
+body {
+  margin: 0;
+}
+
+.block {
+  height: 100px;
+  width: 100%;
+}
+
+.itema {
```

Comment:
In this task ok, but for the future, `itema, itemb, itemc ...` bad name for the classes.